test/python/apps/polkadot.py

```diff
@@ -0,0 +1,151 @@
+import traceback
+from enum import IntEnum
+
+from nacl.encoding import HexEncoder
+from nacl.signing import VerifyKey,SigningKey
+from nacl.exceptions import BadSignatureError
+
+from ragger.utils import create_currency_config
+from ragger.bip import bitcoin_pack_derivation_path, BtcDerivationPathFormat
+from ragger.error import ExceptionRAPDU
+from scalecodec.base import RuntimeConfiguration
+from scalecodec.type_registry import load_type_registry_preset
+from scalecodec.utils.ss58 import ss58_decode
+
+class Method(IntEnum):
+    BALANCE_TRANSFER = 0x0500
+    BALANCE_FORCE_TRANSFER = 0x0502
+
+class AccountIdLookupType(IntEnum):
+    ID = 0
+    INDEX = 1
+    RAW = 2
+    ADDRESS32 = 3
+    ADDRESS20 = 4
+
+def _polkadot_address_to_pk(address: str) -> bytes:
+    return bytes.fromhex(ss58_decode(address))
+
+def _format_amount(amount: int) -> bytes:
+    RuntimeConfiguration().update_type_registry(load_type_registry_preset("legacy"))
+    obj = RuntimeConfiguration().create_scale_object('Compact<Balance>')
+    scale_data = obj.encode(amount)
+    return bytes(scale_data.get_remaining_bytes())
+
+# Not sure what this exactly is but we don't actually care
+UNKNOWN = bytes([0x85, 0x02, 0x00, 0x00])
```

Comment:
Can't we just put a name on what this is ? ^^

Reply:
I would like to, but I spent a lot of time trying to figure out what this does to no avail... Not sure we would gain a lot

Reply:
Ok, no worries, let's go ahead with unknown.